samples/client/petstore/python/tests/test_configuration.py

```diff
@@ -22,14 +22,25 @@ def setUp(self):
         pass
 
     def tearDown(self):
-        pass
+        # reset Configuration
+        petstore_api.Configuration.set_default(None)
 
     def testConfiguration(self):
         # check that different instances use different dictionaries
         c1 = petstore_api.Configuration()
         c2 = petstore_api.Configuration()
-        assert id(c1.api_key) != id(c2.api_key)
-        assert id(c1.api_key_prefix) != id(c2.api_key_prefix)
+        self.assertNotEqual(id(c1.api_key), id(c2.api_key))
+        self.assertNotEqual(id(c1.api_key_prefix), id(c2.api_key_prefix))
+
+    def testDefaultConfiguration(self):
+
+        # prepare default configuration
+        c1 = petstore_api.Configuration(host="example.com")
+        petstore_api.Configuration.set_default(c1)
+
+        # get default configuration
+        c2 = petstore_api.Configuration.get_instance()
+        self.assertEqual(c2.host, "example.com")
```

Comment:
Can you also add a test that verifies that the
contents of the api_key and api_key_prefix dicts are the same but that their ids are different?
Or will they be the same because of the shallow copy?

Reply:
I wasn't aware that api_key are dict, thanks.